index.html

```diff
@@ -1,96 +1,336 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Game Page</title>
-  <link rel="stylesheet" href="/public/css/styles.css" />
-</head>
-<body>
-  <div class="container">
-    <div class="NavBar">
-      <div class="NavBar-Image">
-        <img class="Marvel-Image" src="/Public/marvel.png" alt="Marvel Logo" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Game Page</title>
```

Comment:
Why the title of this page is **Game Page** instead of something related with marvel?

Reply:
👍🏼